wuzz.go

```diff
@@ -495,6 +496,7 @@ func (a *App) SetKeys(g *gocui.Gui) {
 
 	g.SetKeybinding("", gocui.KeyCtrlR, gocui.ModNone, a.SubmitRequest)
 	g.SetKeybinding("url", gocui.KeyEnter, gocui.ModNone, a.SubmitRequest)
+	g.SetKeybinding("", gocui.KeyCtrlU, gocui.ModNone, a.ExportAsCURLCommand)
```

Comment:
Ctrl+U is not a reserved key of the shell. It is a key configured for the whole terminal and any application running inside it. See `stty -a`. On my terminal Ctrl+U is also attached to the `kill` function of the terminal.